include/crow/http_server.h

```diff
@@ -187,12 +187,12 @@ namespace crow
 
         void signal_clear()
         {
-            signals_.clear();
+            //signals_.clear();
```

Comment:
You can leave this line uncomment. It will not do anything. (it will clear an empty vector).